inference-engine/src/low_precision_transformations/src/rt_info/intervals_alignment_attribute.cpp

```diff
@@ -0,0 +1,217 @@
+    const AttributeParameters& params) {
+    if (!is_type<opset1::FakeQuantize>(node)) {
+        return nullptr;
+    }
+
+    auto fakeQuantize = as_type_ptr<opset1::FakeQuantize>(node);
+    if (!QuantizationDetails::outputLayoutIsSupported(fakeQuantize) || !QuantizationDetails::isSupportedLevel(fakeQuantize->get_levels())) {
+        return nullptr;
+    }
+
+    float lowInterval;
+    float highInterval;
+    {
+        OV_ITT_SCOPE(FIRST_INFERENCE, itt::domains::LPT_LT, "calculateIntervals");
+
+        FakeQuantizeDequantization dequantization;
+        {
+            const auto targetInputs = node->output(0).get_target_inputs();
+            if (targetInputs.size() == 1ul) {
+                dequantization = NetworkHelper::getDequantizationBelow(node, true);
+            }
+        }
+
+        const auto outLow = as_type_ptr<opset1::Constant>(node->get_input_node_shared_ptr(3));
+        const auto outHigh = as_type_ptr<opset1::Constant>(node->get_input_node_shared_ptr(4));
+        if (!NetworkHelper::isScalarLike(outLow) || !NetworkHelper::isScalarLike(outHigh)) {
+            return nullptr;
+        }
+
+        if (dequantization.empty()) {
+            const std::vector<float> lowIntervals = outLow->cast_vector<float>();
+            lowInterval = *std::min_element(lowIntervals.begin(), lowIntervals.end());
+
+            const std::vector<float> highIntervals = outHigh->cast_vector<float>();
+            highInterval = *std::max_element(highIntervals.begin(), highIntervals.end());
+        } else {
+            {
+                auto multiplyResult = dequantization.multiplyConstant == nullptr ?
+                    node->get_input_node_ptr(3)->shared_from_this() :
+                    fold<opset1::Multiply>(
+                        foldConvert(node->get_input_node_ptr(3)->shared_from_this(), params.deqPrecision),
+                        dequantization.multiplyConstant);
+
+                auto multiplyResultConstant = as_type_ptr<opset1::Constant>(multiplyResult);
+                auto intervals = multiplyResultConstant->cast_vector<float>();
+                lowInterval = *std::min_element(intervals.begin(), intervals.end());
+            }
+
+            {
+                auto multiplyResult = dequantization.multiplyConstant == nullptr ?
+                    node->get_input_node_ptr(4)->shared_from_this() :
+                    fold<opset1::Multiply>(
+                        foldConvert(node->get_input_node_ptr(4)->shared_from_this(), params.deqPrecision),
+                        dequantization.multiplyConstant);
+
+                auto multiplyResultConstant = as_type_ptr<opset1::Constant>(multiplyResult);
+                auto intervals = multiplyResultConstant->cast_vector<float>();
+                highInterval = *std::max_element(intervals.begin(), intervals.end());
+            }
+        }
+
+        if (std::isinf(lowInterval) || std::isinf(highInterval)) {
+            // ticket: 57172
```

Comment:
It seems, this ticket is closed as not a bug.

Reply:
done, thanks